lib/bazel/exec.go

```diff
@@ -57,10 +58,25 @@ var NewCommand = func(cmd *exec.Cmd) (Command, error) {
 		return nil, fmt.Errorf("failed to create stderr file: %w", err)
 	}
 	stderr.Close()
+
+	// BUG(INFRA-140) - By default, cargo will download packages to a well-known
```

Comment:
does this affect local builds as well? Should we change it in the bazel rules themselves rather than in the runner?